foundryup/foundryup

```diff
@@ -252,6 +259,39 @@ OPTIONS:
 EOF
 }
 
+list() {
+  if [ -d "$FOUNDRY_VERSIONS_DIR" ]; then
+    for VERSION in $FOUNDRY_VERSIONS_DIR/*; do
+      say "${VERSION##*/}"
+      for bin in "${BINS[@]}"; do
+        bin_path="$VERSION/$bin"
+        say "- $(ensure "$bin_path" --version)"
```

Comment:
if possible, would be nice for the last item to append `\n` readability as in:

```
foundryup: nightly
foundryup: - forge 0.2.0 (2f698e4 2024-12-13T06:35:25.027879296Z)
foundryup: - cast 0.2.0 (2f698e4 2024-12-13T06:35:25.025215009Z)
foundryup: - anvil 0.2.0 (2f698e4 2024-12-13T06:35:25.023959451Z)
foundryup: - chisel 0.2.0 (2f698e4 2024-12-13T06:35:25.026724926Z)

foundryup: nightly-e5dbb7a320c2b871c4a4a1006ad3c15a08fcf17b
foundryup: - forge 0.2.0 (e5dbb7a 2024-12-03T00:24:42.577356416Z)
foundryup: - cast 0.2.0 (e5dbb7a 2024-12-03T00:24:42.537035487Z)
foundryup: - anvil 0.2.0 (e5dbb7a 2024-12-03T00:24:42.571016376Z)
foundryup: - chisel 0.2.0 (e5dbb7a 2024-12-03T00:24:42.529565979Z
```

Reply:
yep, added in https://github.com/foundry-rs/foundry/pull/9551/commits/bf24bd9155a42c152d290771f0a0496ada8ee46a